src/nhex_bass_hit.cpp

```diff
@@ -27,33 +27,30 @@
 #include <Arduino.h>
 #include "./nhex_bass_hit.h"
 
-extern "C" {
-extern const int16_t AudioWaveformSine[257];
+extern "C"
+{
+  extern const int16_t AudioWaveformSine[257];
 }
 
 void NHexBassHit::noteOn(void)
 {
   __disable_irq();
-
-  wav_phasor = 0;
-  wav_phasor2 = 0;
```

Comment:
These two lines made the nasty click. there is no real reason to bring the phase of the two oscillators to zero on retrig.. In analog circuitry, the oscillators don't do this unless a special function makes them do it on purpose.